dataclass_wizard/parsers.py

```diff
@@ -433,7 +433,7 @@ def __post_init__(self, cls: Type,
                       get_parser: GetParserType):
 
         # Get the field annotations for the `NamedTuple` type
-        type_anns: Dict[str, Type[T]] = get_named_tuple_field_types(
+        type_anns: Dict[str, Type[Any]] = get_named_tuple_field_types(
```

Comment:
As i understand, annotations are the specified field types and there is no bound for their types

Reply:
agreed 💯 . I guess I wasn't looking at it too closely when I decided to subscript like `Type[T]`. Or maybe I just copied what I was using for most things; but you're totally right on this one.